Gems/Prefab/PrefabBuilder/PrefabGroup/ProceduralAssetHandler.cpp

```diff
@@ -68,6 +79,84 @@ namespace AZ::Prefab
         extensions.push_back(PrefabGroupAssetHandler::s_Extension);
     }
 
+    void PrefabGroupAssetHandler::AssetTypeInfoHandler::AddContextMenuActions(
+        [[maybe_unused]] QWidget* caller,
+        QMenu* menu,
+        const AZStd::vector<AzToolsFramework::AssetBrowser::AssetBrowserEntry*>& entries)
+    {
+        using namespace AzToolsFramework::AssetBrowser;
+        auto entryIt = AZStd::find_if
+        (
+            entries.begin(),
+            entries.end(),
+            [](const AssetBrowserEntry* entry) -> bool
+            {
+                return entry->GetEntryType() == AssetBrowserEntry::AssetEntryType::Product;
+            }
+        );
```

Comment:
Is this going to add an entry to every product in the asset browser? Do we need to filter it to just procprefabs?

Reply:
line 105 should prevent each product asset getting this option

Reply:
Oh, I missed that, looks good then